src/components/forms/FileInput/FileInput.stories.tsx

```diff
@@ -158,3 +158,15 @@ export const withRefAndCustomHandlers = (
     </>
   )
 }
+
+export const customText = (): React.ReactElement => (
+  <FormGroup>
+    <Label htmlFor="file-input-single">La entrada acepta un solo archivo</Label>
+    <FileInput 
+      id="file-input-single" 
+      name="file-input-single" 
+      dragText="Arrastre el archivo aquí o "
+      chooseText="elija de una carpeta"
```

Comment:
@werdnanoslen FYI There's also error text - still a bit more english in this component looks for  _"This is not a valid file type."_.  I just saw this.